config/owasp/suppressions.xml

```diff
@@ -6,18 +6,4 @@
 <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-crypto@.*$</packageUrl>
 <cve>CVE-2018-1258</cve>
 </suppress>
-   <suppress>
-    <notes><![CDATA[
-   file name: tomcat-embed-core-9.0.38.jar
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
-    <cve>CVE-2020-13943</cve>
-  </suppress>
-  <suppress>
-    <notes><![CDATA[
-   file name: tomcat-embed-websocket-9.0.38.jar
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-    <cve>CVE-2020-13943</cve>
-  </suppress>
```

Comment:
remove  entry from confluence if done while adding this CVE

Reply:
removed alredy :)